### PR TITLE
[Test] Fix temp folder coverage

### PR DESCRIPTION
**Pull request by franklupo**
Use temp folder user `%USERPROFILE%\AppData\Local\Temp`

### CODE REVIEW

**Comment by dvoituron** [comment]
Why this request?
What is the advantage to use `%USERPROFILE%\AppData\Local\Temp` and not `C:\Temp` (shorter)?

File: tests/Core/_StartCodeCoverage.cmd [hunk 1]
```diff
@@ -18,6 +18,8 @@ REM     $:\> explorer C:\Temp\Coverage\index.html
 echo on
 cls
 
+mkdir %USERPROFILE%\AppData\Local\Temp\FluentUI\Coverage
```
Comment (dvoituron) [inline tests/Core/_StartCodeCoverage.cmd:21]: You don't need to create the folder. This will be done with the `reportgenerator` command.

**Review by dvoituron — CHANGES_REQUESTED** [review]

**Comment by franklupo** [comment]
normally it does not exist, and for safety the directory on c:\ is not recommended

**Comment by vnbaaij** [comment]
> normally it does not exist, 

As @dvoituron says, it gets created automatically.

> and for safety the directory on c:\ is not recommended

On a general level I agree, but this is only ever being used by developers that are contributing to and writing test code for this specific repository. So very aware and highly professional individuals that understand what is happening. We don't need to burie the (temp) result that deep in the file system.

**Pull request closed without merging**